src/api/controllers/articlesController.js

```diff
@@ -0,0 +1,66 @@
+import model from '../models/index';
+import status from '../../helpers/constants/status.codes';
+import errorMessage from '../../helpers/constants/error.messages';
+import errorSender from '../../helpers/error.sender';
+/**
+ * @class
+ */
+export default class Articles {
+  /**
+   * This function gets all the articles that are stored in the article table
+   *
+   * @static
+   * @param {*} req the request
+   * @param {*} res the response to be sent
+   * @memberof Articles
+   * @returns {Object} res
+   */
+  static async getArticles(req, res) {
+    const { Article } = model;
+    const { page } = req.query;
+    const articles = await Article.findAndCountAll();
```

Comment:
@truestbyheart why do you have two queries here?

`articles` and `pagedArticles`??

Reply:
``articles`` is fetching all articles with row count, while ```pagedArticle``` returns the filtered articles according to the offset and limits and page.

Reply:
@truestbyheart no need for two queries. We can use findAndCountAll in addition with limit and offset 

See here: http://docs.sequelizejs.com/manual/models-usage.html